server/models/handlers/appointment.js

```diff
@@ -24,7 +24,18 @@ async function getAppointmentDetails(studentId, workerId, status) {
     return appointmentModel.getAppointmentDetails(studentId, workerId, status);
 }
 
+// Attempts to cancel all the worker appointments and updates their availability to unavailable for the entire day
+// or for a specified period.
+// Returns true if the worker appointments and their times were successfully updated and false if error encountered.
+async function cancelWorkerAppointments(workerId, specificDate) {
```

Comment:
Hi Amy,
Who can cancel appointments for a given s `workerId `? If it is only the worker in question, then we should pass an accessToken to ensure that the specified worker is the only person who is able to cancel appointments for given `specificDate` and their given `workerId `.

Reply:
Hi Nancy,
I agree and that's the reasoning behind implementing this endpoint in the `private.js` file. On the top of the `private.js` file, the router is already configured to check the accessToken for all private endpoints. Thus, this has already been accounted for. If you performed testing with Postman, this would also be visibly apparent as an error would be thrown if an accessToken is not passed in.

Reply:
Hi Amy, 
I'm using the following parameters in Postman:
<img width="944" alt="Screen Shot 2020-11-12 at 12 47 58 PM" src="https://user-images.githubusercontent.com/71143782/98976496-6aa15200-24e5-11eb-8612-5d805f0e444d.png">

I believe this error may be because I am using the incorrect GET vs POST. Could you clarify which I should use?

Reply:
Hi Nancy,
As mentioned in the description of the PR, the endpoint is `api/cancel-specific-day`. Your screenshot shows `cancelWorkerAppointments` instead of the appropriate endpoint.

Reply:
Thanks for catching that! Postman returns the expected output for the worker ID: 8000000, specificDate: 2020-12-20 and accessToken eeJAQr3wEC6CJZROFJTY.